app/controllers/application_controller.rb

```diff
@@ -123,7 +123,7 @@ def set_app_info_headers
     headers['X-Git-SHA'] = AppInfo::GIT_REVISION
   end
 
-  def authenticate
+  def authenticate    
```

Comment:
Looks like a couple spaces mistakenly snook in here.  FWIW, you can add the below to your sublime settings to mitigate that:

```
"ensure_newline_at_eof_on_save": true,
"trailing_spaces_trim_on_save": true,
"translate_tabs_to_spaces": true,
"trim_trailing_white_space_on_save": true,
```